cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -566,9 +566,9 @@ func (uc *UpstreamController) queryService(stop chan struct{}) {
 					log.LOGGER.Warnf("message: %s process failure, response failed with error: %s", msg.GetID(), err)
 					continue
 				}
-				log.LOGGER.Warnf("message: %s process successfully", msg.GetID())
 			default:
 				log.LOGGER.Infof("message: %s process failure, service operation: %s unsupported", msg.GetID(), msg.GetOperation())
+				continue
```

Comment:
Maybe the continue don't needed here?

Reply:
If delete this line, will get such log lines:
```
message: xxx process failure, service operation: xxx unsupported
message: xxx process successfully
```
So I think it should not be deleted

Reply:
Agree with sid, delete L573. After delete L573, why will get such log lines?
```
message: xxx process failure, service operation: xxx unsupported
message: xxx process successfully
```

Reply:
Sorry, this is a misunderstanding, I mistakenly think that you want to delete continue

Reply:
Np :-)